### PR TITLE
12 accommodation page fav

### PR DESCRIPTION
**Pull request by RobinDela**

### CODE REVIEW

File: models/favorites.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  select accomodations.*, (
+		select url from photos where idAccomodation=accomodations.id limit 1
+	) as url
+from 
+	favorites
+    join accomodations on favorites.idAccomodation=accomodations.id
+    join photos on photos.idAccomodation=accomodations.id
+where
+	favorites.idUser=?
+group by id
```
Comment (bdtkevin) [inline models/favorites.js:14]: Strange indentation 🤣

**Review by bdtkevin — CHANGES_REQUESTED** [review]

**Review by bdtkevin — DISMISSED** [review]
👍

**Comment by LoicBrassart** [comment]
Please solve conflicts

**Review by bdtkevin — DISMISSED** [review]

File: models/favorites.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const { db } = require('../conf');
+
+const getFav = (id) => {
+  const sql = `
+  select accomodations.*, (
```
Comment (LoicBrassart) [inline models/favorites.js]: Don't use "SELECT *"

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Stop using "SELECT *"

**Comment by LoicBrassart** [comment]
No validation from the SM

**Review by bdtkevin — CHANGES_REQUESTED** [review]
SELECT *

**Review by bdtkevin — APPROVED** [review]
👍🏼👍🏼

**Review by LoicBrassart — APPROVED** [review]
NmE

**Pull request merged**